scipy/spatial/transform/rotation.py

```diff
@@ -1465,6 +1466,58 @@ def magnitude(self):
         else:
             return angles
 
+    def mean(self, weights=None):
+        """Get the mean of the rotations.
+
+        Parameters
+        ----------
+        weights : array_like shape (N,), optional
+            Weights describing the relative importance of the rotations. If
+            None (default), then all values in `weights` are assumed to be
+            equal.
+
+        Returns
+        -------
+        mean : `Rotation` instance
+            Object containing the mean of the rotations in the current
+            instance.
+
+        Notes
+        -----
+        The mean used is the chordal L2 mean (also called the projected or
+        induced arithmetic mean). If ``p`` is a set of rotations with mean
+        ``m``, then ``m`` is the rotation which minimizes
+        ``(weights[:, None, None] * (p.as_dcm() - m.as_dcm())**2).sum()``.
```

Comment:
I updated this line --- weights should multiply the squared difference (checked with another paper).

Reply:
What a silly mistake.